contracts/modules/LidoModule.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import '@yield-protocol/vault-interfaces/IJoin.sol';
+import '@yield-protocol/vault-interfaces/ICauldron.sol';
+import '@yield-protocol/utils-v2/contracts/token/IERC20.sol';
+import '@yield-protocol/utils-v2/contracts/token/TransferHelper.sol';
+import '@yield-protocol/utils-v2/contracts/math/WMul.sol';
+import '@yield-protocol/utils-v2/contracts/cast/CastU256U128.sol';
+import '@yield-protocol/utils-v2/contracts/cast/CastU256I128.sol';
+import '@yield-protocol/utils-v2/contracts/cast/CastU128I128.sol';
+import '../LadleStorage.sol';
+import '../oracles/lido/IWstETH.sol';
+
+/// @dev A ladle module to handle wrapping & unwrapping of stETH
+contract LidoModule is LadleStorage {
+    using WMul for uint256;
+    using CastU256U128 for uint256;
+    using CastU256I128 for uint256;
+    using CastU128I128 for uint128;
+    using TransferHelper for IERC20;
+    using TransferHelper for IWstETH;
+    using TransferHelper for IWETH9;
+    using TransferHelper for address payable;
+    IWstETH public wstETH;
```

Comment:
```suggestion
    IWstETH public immutable wstETH;
    IERC20 public immutable stETH;
```

State variables in modules need to be `immutable` to be accessible from within the module. This is because they then get stored in the bytecode, same as functions. Non-immutable state variables get stored in storage, which is not accessible from within a module (because we are using the storage of the contract using `delegatecall`, the Ladle).